tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "forceConsistentCasingInFileNames": true,
     "noEmit": true,
     "esModuleInterop": true,
-    "module": "esnext",
+    "module": "commonjs",
```

Comment:
Curious as to what prompted the change to commonjs from esnext.

Reply:
Good question! I was running into the following error trying to run `npx prisma db seed`.
It looks like the seed command utilizes ts-node, which requires commonjs from my understanding.
https://github.com/TypeStrong/ts-node/issues/922
https://stackoverflow.com/questions/58273824/typescript-cannot-use-import-statement-outside-a-module